app/scripts/views/sign_up.js

```diff
@@ -192,7 +193,10 @@ function (Cocktail, _, p, BaseView, FormView, Template, Session, AuthErrors,
     },
 
     suggestEmail: function () {
-      mailcheck(this.$el.find('.email'), this.metrics, this.translator, this.window.location.search);
+      console.log(this._able);
+      console.log(this._able.choose('mailcheckEnabled'));
```

Comment:
@dannycoates this is always returning `false`, see https://github.com/mozilla/fxa-content-experiments/pull/5/files

Any idea why?